src/bindings/c/tests/ov_compiled_model_test.cpp

```diff
@@ -145,6 +145,36 @@ TEST_P(ov_compiled_model_test, get_property) {
     ov_core_free(core);
 }
 
+TEST_P(ov_compiled_model_test, set_property) {
+    auto device = GetParam();
+    std::string device_name = "BATCH:" + device + "(4)";
+    ov_core_t* core = nullptr;
+    OV_EXPECT_OK(ov_core_create(&core));
+    EXPECT_NE(nullptr, core);
+
+    ov_model_t* model = nullptr;
+    OV_EXPECT_OK(ov_core_read_model(core, xml_file_name.c_str(), bin_file_name.c_str(), &model));
+    EXPECT_NE(nullptr, model);
+
+    ov_compiled_model_t* compiled_model = nullptr;
+    OV_EXPECT_OK(ov_core_compile_model(core, model, device_name.c_str(), 0, &compiled_model));
+    EXPECT_NE(nullptr, compiled_model);
+
+    const char* key = ov_property_key_auto_batch_timeout;
+    char* result = nullptr;
+
+    OV_EXPECT_OK(ov_compiled_model_get_property(compiled_model, key, &result));
+    EXPECT_STREQ("1000", result);  // default value from /src/plugins/auto_batch/src/plugin.cpp
```

Comment:
I supposed we don't need check its default value, but need set a different value in below set_property() test.